fundingmanager.go

```diff
@@ -2429,6 +2435,11 @@ func (f *fundingManager) handleFundingLocked(fmsg *fundingLockedMsg) {
 		return
 	}
 
+	// As this is the earliest possible time that the channel can be used
+	// to create/receive commitment updates, we'll mark the channel has
+	// active now.
+	f.cfg.NotifyReadyChannelEvent(channel.FundingOutpoint)
```

Comment:
reminder that with #3369 this is no longer true, it should be done from inside the link